meson.build

```diff
@@ -20,7 +20,7 @@ project(
 # - reset minor version to zero when major version is incremented
 va_api_major_version = 1
 va_api_minor_version = 21
-va_api_micro_version = 0
+va_api_micro_version = 1
```

Comment:
please dont change it, 
our current release process is:
master: x.xx.0.pre1 
then release as x.xx.0
then bump master to x."xx+1".0.pre

if there are hotfix in release branch , we will bump micro version

Reply:
Thanks for the explanation, updated without the version change.